OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -102,6 +102,8 @@ public OpenAlSoundEngine()
 					throw new InvalidOperationException("Can't create OpenAL device");
 			}
 
+			Game.OnQuit += () => { ALC10.alcCloseDevice(device); };
```

Comment:
This shouldn't be using OnQuit at all. That's a legacy hack from before we started treating object lifetimes properly.